lib/typescript/httpclient/endpoints/updateTag.ts

```diff
@@ -0,0 +1,12 @@
+import {doFetchFromBackend} from '../api';
+import {Tag} from '../model';
+
+export function updateTag(tag: Tag) {
+  return doFetchFromBackend('tags.update', {
+    id: tag.id,
+    name: tag.name,
+    color: tag.color,
+  })
```

Comment:
(honest question) would `{...tag}` would here?

Reply:
yesss the tag object has exactly the same format so it works!